modules/apm/variables.tf

```diff
@@ -0,0 +1,69 @@
+# Global Terraform
+variable "environment" {
+  description = "Architecture environment"
```

Comment:
I find the term _Architecture_ a bit inappropriate in this context. What about _Application_ or _Platform_? Or maybe just _Environment_?

Reply:
TBO I just copy-pasted this line, so yeah I guess _Application_ makes more sense in the current context 👍